src/vos/evtree.c

```diff
@@ -2578,14 +2578,20 @@ evt_ent_array_fill(struct evt_context *tcx, enum evt_find_opc find_opc,
 				 * place so I did it this way to minimize
 				 * change while we decide how to handle this
 				 * properly.
+				 * RT_OVERLAP_INCLUDED/RT_OVERLAP_INCLUDES is possible
+				 * when vos aggregation merged exts between retried
+				 * rebuild, returned DER_RECX_OVERLAP for that case.
```

Comment:
When the recx being insert is larger than the in-tree recx (RT_OVERLAP_INCLUDES case), I think we can't simply ignore it, otherwise, we could miss the larger portion, right?

If current tests failed only for RT_OVERLAP_INCLUDED case, I think we could just amend current patch to ignore the RT_OVERLAP_INCLUDED case only, otherwise, I think we might have to introduce a new vos update API used by rebuild internally, the API should be able to return detailed information (the missed portion) for rebuild to retry, what do you think?

Reply:
"If current tests failed only for RT_OVERLAP_INCLUDED case"
yes, current test failed all belong to this case. it it the case that local vos agg merge exts between retired rebuild.

Why I also added RT_OVERLAP_INCLUDES is considering another possible case - retry rebuild from beginning enumerate from remote parity shard it got merged exts [0-2M] then rebuild it, but locally the vos agg has not merge [0-1M] and [1M-2M], then it possibly will hit RT_OVERLAP_INCLUDES, although it is not the case in failed CI test.

I'm fine to change it only for RT_OVERLAP_INCLUDED case, if that is safer.

Reply:
I'll change to only handle RT_OVERLAP_INCLUDED for now, for RT_OVERLAP_INCLUDES case not sure if really will hit that case can handle that later if needed.

Reply:
"reclaim" phase probably can be done in another patch.
"also needs to understand why the initial rebuild fails." some info in DAOS-10435, we can chat offline for some more details.